userguide/platform/userguide_configuration.adoc

```diff
@@ -212,13 +212,116 @@ The following table lists some of the Kill Bill configuration properties.
+|Security   |org.killbill.security.okta.apiToken   |KB_org_killbill_security_okta_apiToken   |Okta API token   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.okta.permissionsByGroup   |KB_org_killbill_security_okta_permissionsByGroup   |Okta permissions by Okta group   |admin = *:*\n" +
+             "finance = invoice:*, payment:*\n" +
+             "support = entitlement:*, invoice:item_adjust   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.url   |KB_org_killbill_security_auth0_url   |Auth0 tenant full url   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.clientId   |KB_org_killbill_security_auth0_clientId   |Auth0 application Client ID   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.clientSecret   |KB_org_killbill_security_auth0_clientSecret   |Auth0 application Client Secret   |Null   |Config File/Environment Variable
+|Security   |org.killbill.security.auth0.apiIdentifier   |KB_org_killbill_security_auth0_apiIdentifier   |Auth0 API identifier   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.issuer   |KB_org_killbill_security_auth0_issuer   |Auth0 JWT expected issuer   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.audience   |KB_org_killbill_security_auth0_audience   |Auth0 JWT expected audience   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.usernameClaim   |KB_org_killbill_security_auth0_usernameClaim   |JWT claim to use as the user name   |sub   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.databaseConnectionName   |KB_org_killbill_security_auth0_databaseConnectionName   |Auth0 database connection name   |Null   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.connectTimeout   |KB_org_killbill_security_auth0_connectTimeout   |Auth0 client connect timeout   |5s   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.readTimeout   |KB_org_killbill_security_auth0_readTimeout   |Auth0 client read timeout   |60s   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.requestTimeout   |KB_org_killbill_security_auth0_requestTimeout   |Auth0 client request timeout   |60s   |Config File/Environment Variable   
+|Security   |org.killbill.security.auth0.allowedClockSkew   |KB_org_killbill_security_auth0_allowedClockSkew   |Auth0 JWT allowed clock skew   |60s   |Config File/Environment Variable   
+|Security   |org.killbill.security.skipAuthForPlugins   |KB_org_killbill_security_skipAuthForPlugins   |Specifies whether authentication should be skipped for plugins   |false   |Config File/Environment Variable   
 |Security   |killbill.server.ldap   |-   |Coming Soon   |false   |System property   
```

Comment:
The `org.killbill.security.skipAuthForPlugins` property is not a part of a `config` class like other properties. If there are any other properties that are not part of any config classes, could you please point me to them, so that I can add those to the document as well?

Reply:
There are very few config properties that are not part of a config class; the one you mention is defined in [PermissionAnnotationMethodInterceptor](https://github.com/killbill/killbill/blob/master/util/src/main/java/org/killbill/billing/util/security/PermissionAnnotationMethodInterceptor.java#L35). Typically, when we do that this is precisely because we don't want those to be 'too' documented... 

/cc @pierre Any thoughts?

Reply:
True, although for this one specifically, we want it to be documented 😄  

For the other ones, let's look at them on a case by case basis, as they come up in support discussions in the future.